src/styles/styles.js

```diff
@@ -47,15 +47,37 @@ const styles = {
     h3: {
         fontFamily: fontFamily.GTA_BOLD,
         fontSize: variables.fontSizeNormal,
+    },
+
+    h3Bold: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: variables.fontSizeNormal,
         fontWeight: fontWeightBold,
     },
 
     h4: {
         fontFamily: fontFamily.GTA_BOLD,
         fontSize: variables.fontSizeLabel,
+    },
+
+    h4Bold: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: variables.fontSizeLabel,
         fontWeight: fontWeightBold,
     },
 
+    fontFamilyGTA: {
```

Comment:
This style shouldn't be necessary, it is likely that it's being added to `Text` components that are included from `react-native` instead of `components/Text`. Could you update any places you're using this style to ensure the `Text` component is being imported from `components/Text` instead of `react-native`?